dev-docs/makefile.md

```diff
@@ -6,14 +6,18 @@ The Makefile `tasks.mak` is a Makefile for the project that is used to quickly p
 
 We are unable to use the actual `Makefile` name in this repo; because we build the Ops Agent into a deb package, `debhelper` tries to autodetect `make` as the build system in most of its `dh_auto` steps. We do not want `debhelper` to pick up the Makfile and try picking up different targets to build and test it.
 
+## Disclaimer
+
+`tasks.mak` is provided without any guarantees or warranty on its targets. It is meant purely for developer convenience, and it is advised not to make any dependency on the targets since they are subject to change at any time.
```

Comment:
[Optional] I wonder if we should be calling it `developer_tasks.mak` or something…

Reply:
I originally chose `tasks.mak` just cause it was shorter, and at first it seemed like we'd need to always type in `-f tasks.mak` so making it longer would have been a pain. We could change it now, though it really only exists to then be immediately symlinked to when you set up your environment.